core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -130,13 +130,8 @@ public FileOutStream createFile(AlluxioURI path, CreateFilePOptions options)
     try {
       masterClient.createFile(path, options);
       // Do not sync before this getStatus, since the UFS file is expected to not exist.
-      GetStatusPOptions gsOptions = FileSystemClientOptions.getGetStatusOptions();
-      gsOptions = gsOptions.toBuilder()
-          .setLoadMetadataType(LoadMetadataPType.NEVER)
-          .setCommonOptions(
-              FileSystemClientOptions.getCommonOptions().toBuilder()
-                  .setSyncIntervalMs(-1))
-          .build();
+      GetStatusPOptions gsOptions =
+          GetStatusPOptions.newBuilder().setLoadMetadataType(LoadMetadataPType.NEVER).build();
```

Comment:
Don't need the sync interval option anymore?

Reply:
-1 is the default value. That's why it's deleted here.
Going forward you don't need to set anything that's default.